tests/test_api.py

```diff
@@ -6,7 +6,13 @@
 import os
 import pandas as pd
 import re
+from math import ceil
+import logging
+from time import perf_counter
+from schematic.schemas.generator import SchemaGenerator
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
When I ran `class TestValidationBenchmark`, I could see a lot of Debug messages related to error of manifest. I understand this is expected because the length of invalid manifest is over 1000 lines. But is it possible to modify that a bit so that you don't get crazy log when running `class TestValidationBenchmark`